src/qtcreatorlibrary.pri

```diff
@@ -28,9 +28,12 @@ CONFIG += shared dll
 
 contains(QT_CONFIG, reduce_exports):CONFIG += hide_symbols
 
-win32|os2 {
+win32 {
```

Comment:
See above.

Reply:
because dlltarget seems not be supported for os2, I see it only in win32 qmake code.